src/templates/ProjectPageEN.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { LocaleProvider, NavbarStateProvider } from '../utils/hooks'
+import Layout from '../components/layout'
+import { ProjectPageComponent } from '../components/PastProjects/Page'
+
+export default ({ pageContext }) => (
+  <LocaleProvider value="en">
+    <NavbarStateProvider>
+      <Layout {...pageContext.head}>
+        <ProjectPageComponent {...pageContext} />
+      </Layout>
+    </NavbarStateProvider>
+  </LocaleProvider>
+)
```

Comment:
Why are these template files necessary, can't one just use the components?

Reply:
Maybe you could have gotten away with only writing one of these, and giving it locale as a prop